src/librustc/middle/traits/error_reporting.rs

```diff
@@ -290,7 +290,7 @@ pub fn maybe_report_ambiguity<'a, 'tcx>(infcx: &InferCtxt<'a, 'tcx>,
                     {
                         span_err!(infcx.tcx.sess, obligation.cause.span, E0282,
                                 "unable to infer enough type information about `{}`; \
-                                 type annotations required",
+                                 type annotations or generic parameter binding required",
```

Comment:
hmm, okay, that is a *little* bit more comforting, but still a little worrisome.  Maybe this is something going wrong with our return codes on Windows, causing the `make` to miss the fact that the run errored?